buildspecs/buildspec.release.yml

```diff
@@ -3,17 +3,15 @@ version: 0.2
 env:
   variables:
     AWS_REGION: us-west-2
+    ROLE_ARN: arn:aws:iam::582595803497:role/CloudWatch-Logs-EMF-build-role
```

Comment:
I think we should move this to parameter-store as well. It's probably not a security issue but we should try to avoid exposing as much information as we can.

Reply:
Moved